src/sentry/models/groupassignee.py

```diff
@@ -63,7 +64,8 @@ def assign(self, group, assigned_to, acting_user=None):
                 type=Activity.ASSIGNED,
                 user=acting_user,
                 data={
-                    'assignee': six.text_type(assigned_to.id),
+                    # todo(maxbittker) use actor_ids here?
```

Comment:
Yeah, I was a bit torn on this myself when I added it. This thing is more structured, so I went with `assigneeType` and `assignee` to maintain backwards compat.